components/loading-box/src/stories.js

```diff
@@ -58,7 +61,8 @@ examples.add('preset to loading', () => (
   >
     <div style={{ padding: '0 12px' }}>
       <PhaseBanner level="alpha">
-      This part of GOV.UK is being rebuilt &#8211;{' '}
+      This part of GOV.UK is being rebuilt &#8211;
```

Comment:
indentation here feels wrong too

Reply:
if you don't like eslint-config-airbnb's styling rules then a good argument to move to prettier :-)